docusaurus/docs/reference/ha/operations.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_label: Operations
+sidebar_position: 50
+---
+
+# Operating a Controller Cluster
+
+## Restoring from Backup
+
+To restore from a database snapshot, use the following CLI command:
```

Comment:
I don't first need to put the cluster into a maintenance mode? What if model updates happen? This operation takes precedence?

Reply:
Model updates that happen before the restore will be overwritten. Model updates that happen after the restore will be applied to the restored db, and may fail if they aren't valid.